test/WebJobs.Script.Tests.Integration/TestFunctionHost.cs

```diff
@@ -285,6 +289,18 @@ public void Configure(AspNetCore.Builder.IApplicationBuilder app, AspNetCore.Hos
             }
         }
 
+        private class TestExtensionBundleManager : IExtensionBundleManager
+        {
+            public Task<ExtensionBundleDetails> GetExtensionBundleDetails() => Task.FromResult<ExtensionBundleDetails>(null);
+
+            public Task<string> GetExtensionBundlePath(HttpClient httpClient = null) => null;
```

Comment:
Same here, do not return null. Follow similar: Task.FromResult<ExtensionBundleDetails>(null) and unwrapping task would throw null ref.